hbase-client/src/main/java/org/apache/hadoop/hbase/client/LockTimeoutException.java

```diff
@@ -0,0 +1,32 @@
+/**
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.hbase.client;
+
+import org.apache.hadoop.hbase.HBaseIOException;
+import org.apache.yetus.audience.InterfaceAudience;
+
+/*
+  Thrown whenever we are not able to get the lock within the specified wait time.
+ */
+@InterfaceAudience.Public
```

Comment:
need to be public?

Reply:
Actually I don't know. Since this will thrown back all the way to client so thought to make it public. But open for suggestions.

Reply:
> I just realized there is an exact same class LockTimeoutException under org.apache.hadoop.hbase.exceptions, switch to that?

This class only exists in branch-1 and not in master/branch-2. :(

Reply:
Maybe when I create a new PR for branch-1, I can re-use the existing exception class. Would that work ?